src/js/users/components/Permission.tsx

```diff
@@ -0,0 +1,33 @@
+import React from "react";
+import styled from "styled-components";
+import { BoxGroupSection, Icon } from "../../base";
+
+const StyledPermissionIcon = styled(Icon)`
+    margin-right: 15px;
+`;
+
+const StyledPermissionItem = styled(BoxGroupSection)`
+    align-items: center;
+    display: flex;
+`;
+
+type PermissionItemProps = {
+    /* Name of the permission */
+    permission: string;
+    /* Indicates if the permission is granted */
+    value: boolean;
+};
+
+/**
+ * Permission item with an icon indicating its status.
```

Comment:
```suggestion
 * A condensed permission item for use in a list of user permissions
```

Not sure why but the commit on this didnt apply